tests/fixtures/security/make-form-login/expected/LoginController.php

```diff
@@ -15,13 +15,10 @@ public function login(AuthenticationUtils $authenticationUtils): Response
         // get the login error if there is one
         $error = $authenticationUtils->getLastAuthenticationError();
 
-        // last username entered by the user
+        /** last username entered by the user */
         $lastUsername = $authenticationUtils->getLastUsername();
```

Comment:
php-cs-fixer does not have the capability to add a space *above* a comments (`//`) but it can do it with docblocks (`/** */`)

Without changing the comments over, we would end up with:
```php
        // get the login error if there is one
        $error = $authenticationUtils->getLastAuthenticationError();
        // last username entered by the user
        $lastUsername = $authenticationUtils->getLastUsername();
```

which could be alright?